specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json

```diff
@@ -5577,7 +5577,35 @@
         },
         "enableAutomaticUpdates": {
           "type": "boolean",
-          "description": "Indicates whether virtual machine is enabled for automatic updates."
+          "description": "Indicates whether virtual machine is enabled for automatic windows updates. <br> <br> Default behavior is true."
```

Comment:
Swagger definitions look fine for this. However, there are risks in adding new properties to an existing API version. When updated and non-updated clients have different models for the resource, updates can be ambiguous as to whether a missing property was an intentional removal or simply the result of a downlevel client. Do these need to be added to an existing API version or can they go out in a new version?